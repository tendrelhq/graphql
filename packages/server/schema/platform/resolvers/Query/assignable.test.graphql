query TestAssignable($entity: ID!) {
  assignable(entity: $entity) {
    edges {
      node {
        ... on Worker {
          displayName
        }
      }
    }
  }
}
