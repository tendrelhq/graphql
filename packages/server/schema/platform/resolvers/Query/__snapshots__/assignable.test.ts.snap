// Bun Snapshot v1, https://goo.gl/fbAQLP

exports[`assignable not assignable entity 1`] = `
{
  "data": null,
  "errors": [
    [GraphQLError: Entity cannot be assigned],
  ],
}
`;

exports[`assignable workers 1`] = `
{
  "data": {
    "assignable": {
      "__typename": "AssignableConnection",
      "edges": [
        {
          "__typename": "AssignableEdge",
          "node": {
            "__typename": "Worker",
            "displayName": "Another Guy",
          },
        },
        {
          "__typename": "AssignableEdge",
          "node": {
            "__typename": "Worker",
            "displayName": "Foo 1",
          },
        },
        {
          "__typename": "AssignableEdge",
          "node": {
            "__typename": "Worker",
            "displayName": "Foo 2",
          },
        },
        {
          "__typename": "AssignableEdge",
          "node": {
            "__typename": "Worker",
            "displayName": "How many Guys?",
          },
        },
        {
          "__typename": "AssignableEdge",
          "node": {
            "__typename": "Worker",
            "displayName": "Jalen Hurts",
          },
        },
        {
          "__typename": "AssignableEdge",
          "node": {
            "__typename": "Worker",
            "displayName": "Jason Bourne",
          },
        },
        {
          "__typename": "AssignableEdge",
          "node": {
            "__typename": "Worker",
            "displayName": "New Guy",
          },
        },
        {
          "__typename": "AssignableEdge",
          "node": {
            "__typename": "Worker",
            "displayName": "Randy Moss",
          },
        },
        {
          "__typename": "AssignableEdge",
          "node": {
            "__typename": "Worker",
            "displayName": "Third Guy",
          },
        },
        {
          "__typename": "AssignableEdge",
          "node": {
            "__typename": "Worker",
            "displayName": "Will Ruggiano",
          },
        },
        {
          "__typename": "AssignableEdge",
          "node": {
            "__typename": "Worker",
            "displayName": "Yet Another Brand New Guy",
          },
        },
        {
          "__typename": "AssignableEdge",
          "node": {
            "__typename": "Worker",
            "displayName": "Yet Another New Guy",
          },
        },
      ],
    },
  },
}
`;
